CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
 
 * [#60](https://github.com/nrepl/nrepl/issues/60): Implemented EDN transport.
 * [#140](https://github.com/nrepl/nrepl/issues/140): Added initial version of spec for message responses. These are used during Clojure 1.10 tests.
+
+### Changes
```

Comment:
There's already a section "Changes" for this release.

Reply:
oops. merged.